src/components/Person.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {'My name is '}
+      {name}
+    </h2>
```

Comment:
```suggestion
    <h2 className="Person__name">My name is {name}</h2>
```

Reply:
Linter don't accept this formating ((
he said: `{name}` must be placed on a new line